.github/Contributing.md

```diff
@@ -8,36 +8,36 @@ By submitting code as an individual you agree that VATSIM UK can use your amendm
 
 If you're comfortable with contributing to Open Source projects on GitHub please ensure you read our expectations for issue tracking, feature proposals and merge requests.
 
-**Please avoid** adding airac related commits until you've seen an issue created for them - anything else is fair game.
+**Please avoid** adding AIRAC related commits until you've seen an issue created for them - anything else is fair game.
 
 Please have a thorough read of the 1st-time Contributor's Guide: https://github.com/VATSIM-UK/UK-Sector-File/wiki/First-Time-Contributor's-Guide.
 
 ## Testing the sector file
 
-At present there isn't a publicly accessible auto-generation tool for the data within the repository.  We're working hard to make our current tool stable enough to release into the public domain so that open source contributions can be made to that, too, and others can use (and test) the latest development version.
+For each PR opened within the repository, the Sector File Compiler will generate a 'dev' version of the .sct, .ese and .rwy file for you to test out your changes. The compiler will also tell you if there are any errors. Click 'Details' in the **Checks** section of the Pull Request to check the errors.
 
 ## Issue Tracking
 
-If you require **support** with the Sector File or EuroScope, please utilise our Discord channels or the [VATSIM UK Helpdesk](https://helpdesk.vatsim.uk).  Issues regarding the features and functions of Euroscope or how to load the Sector File will not be handled.  The issue tracker is for feature requests and bugs concerning the UK Sector File itself.
+If you require **support** with the Sector File or EuroScope, please use our Discord channels or the [VATSIM UK Helpdesk](https://helpdesk.vatsim.uk).  Issues regarding the features and functions of EuroScope or how to load the Sector File will not be handled.  The issue tracker is for feature requests and bugs concerning the UK Sector File itself.
 
 When submitting an issue, there's a few guidelines we'd ask you to respect to make it easier to manage (and for others to understand):
 * **Search the issue tracker** before you submit your issue - it may already be present.
 * When opening an issue, a template is provided for you.  Please provide as much information as requested to ensure others are able to act upon the requests or bug report.
 * Please ensure you add screenshots or documentation references for bugs/changes so we can quickly ascertain if the request is suitable.
 
-**In order to be 'assigned' an issue**, please comment on the issue itself - we can then assign your GitHub account to that particular issue.
+**If you can't assign yourself to an issue** please comment on the issue or reach out to us in Discord and we will add you to the repository contributors list so you can assign yourself in future.
```

Comment:
I think we usually only do this after one or more successful PRs? This to me reads as it could/would be done before any PRs, which I find a bit dodgy?